app/models/course.rb

```diff
@@ -1,9 +1,8 @@
 class Course < ApplicationRecord
   belongs_to :user
   has_many :sections
+  has_many :photos
 
-  mount_uploader :photo, PhotoUploader
```

Comment:
The uploader needs to be mounted in the `photo` model if you use this approach (the alternative to what the lessons suggest).